pkg/audit/manager.go

```diff
@@ -274,62 +275,107 @@ func (am *Manager) auditResources(
 	var errs opa.Errors
 	nsCache := newNSCache()
 
-	for gv, gvKinds := range clusterAPIResources {
-		for kind := range gvKinds {
-			objList := &unstructured.UnstructuredList{}
-			opts := &client.ListOptions{
-				Limit: int64(*auditChunkSize),
+	constraintsList := make(map[string]bool)
```

Comment:
It seems to me like a more purpose-oriented name, like "matchedKinds" would be helpful for having the code self-document.

Reply:
done